tests/blackbox_test.go

```diff
@@ -136,54 +136,48 @@ func outer(t *testing.T, test types.Test, negativeTests bool) error {
 					return err
 				}
 			}
-			err := updateTypeGUID(t, part)
+			err := updateTypeGUID(part)
 			if err != nil {
 				return err
 			}
 		}
 
 		disk.SetOffsets()
 		for _, part := range test.Out[i].Partitions {
-			err := updateTypeGUID(t, part)
+			err := updateTypeGUID(part)
 			if err != nil {
 				return err
 			}
 		}
 		test.Out[i].SetOffsets()
 
+		if err = setupDisk(t, ctx, &disk, i, imageSize, tmpDirectory); err != nil {
+			return err
+		}
+
 		// Creation
-		err = createVolume(t, ctx, tmpDirectory, i, disk.ImageFile, imageSize, disk.Partitions)
-		// Move value into the local scope, because disk.ImageFile will change
-		// by the time this runs
+		// Move value into the local scope, because disk.ImageFile and device
+		// will change by the time this runs
 		imageFile := disk.ImageFile
+		device := disk.Device
 		defer func() {
-			err := os.Remove(imageFile)
-			if err != nil {
+			t.Logf("removing file")
```

Comment:
Is this additional log output accidental?

Reply:
Accidental